scripts/lib/sharding.py

```diff
@@ -93,7 +93,7 @@ def write_updated_configs() -> None:
 full_new_paths = [f"{filename}.tmp" for filename in full_real_paths]
 try:
     write_updated_configs()
-    for old, new in zip(full_real_paths, full_new_paths):
+    for old, new in zip(full_real_paths, full_new_paths, strict=False):
```

Comment:
Do we want `strict=False`? I feel like probably ~all of these code paths expect the two lists to have the same length, and `strict=True` would be the correct approach.

Reply:
I think so too, but wanted to separate out that potential behavior change from these syntax changes.